presto-main/src/test/java/com/facebook/presto/execution/buffer/BufferTestUtils.java

```diff
@@ -120,6 +122,15 @@ static void acknowledgeBufferResult(OutputBuffer buffer, OutputBuffers.OutputBuf
         return future;
     }
 
+    public static void addPages(OutputBuffer buffer, List<Page> pages)
+    {
+        List<SerializedPage> serializedPages = new LinkedList<>();
+        for (Page page : pages) {
```

Comment:
I know this is a test utility but it would be nice to have a null filter for pages.